pyproject.toml

```
@@ -46,7 +47,7 @@ classifiers = [
 ]
 dependencies = [
   "packaging>=17.1",
-  "pytest>=7.2",
+  "pytest>=7.4",
```

```suggestion
  "pytest>=7.4,!=8.2.2",
```

as per https://github.com/pytest-dev/pytest-rerunfailures/issues/267#issuecomment-2457891221